data/pc/1.21.8/blocks.json

```diff
@@ -2703,7 +2703,9 @@
         "num_values": 2
       }
```

Comment:
Looks like only change in .8 is block drops, the entities.json changed because the entity metadata extractor was not run in other PR